helm-chart/renku-graph/charts/jena/templates/configmap.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jena-shiro-ini
+  labels:
+    app: {{ template "jena.name" . }}
+    chart: {{ template "jena.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  shiro.ini: |-
```

Comment:
this might be better as a secret instead of configmap since it contains credentials?

Reply:
I guess it would be better (and there is a secret to be used for microservice's configs) but Jena itself requires a `shiro.ini` file in her folder with the creds in open text.

Reply:
you can mount a secret into a file all the same

Reply:
e.g. https://kubernetes.io/docs/concepts/configuration/secret/#use-case-pod-with-ssh-keys

Reply:
Do you have an example somewhere? I just can't think of how this should be looking like :)

Reply:
But wait there are two things I'm not sure about here:
1) the `shiro.ini` is generated with the values from the config and mounted directly into jena's filesystem so the configmap.yaml doesn't contain the passwords just the placeholders - what we would gain from turning it into a secret?
2) we need these passwords to be in both the `shiro.ini` and the 'normal' secrets.yaml so they can be injected into microservices' environment - how you can use the same values in both places? Can you have placeholders in a secret file?

Reply:
> what we would gain from turning it into a secret?
true, atm there isn't much of a difference, except that in certain APIs secrets are hidden and only available to a user with sufficient privileges. However it's a best-practice to use secrets for confidential information. 

You can basically use exactly the same configuration you have here but make it a secret instead of a configmap. There are some differences in mounting, I believe you can look at how we mount the keycloak realm if you want an example.

Reply:
Ok, let me try that

Reply:
It worked!